### PR TITLE
Enhancement: Pass Wallet to More Translate Functions in Connect Interface

### PR DESCRIPTION
**Pull request by leightkt**
### Description
Pass wallet as a value to more areas of the translate function in the connect interface to allow for further language customization per https://github.com/blocknative/web3-onboard/issues/1632

Screenshot 2023-04-21 at 1 20 36 PM

Screenshot 2023-04-21 at 1 20 27 PM

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [x] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [x] Add/update the appropriate package README (if applicable)
- [NA ] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [ NA] Add/update the related package in the `docs/package.json` file (if applicable)

Screenshot 2023-04-21 at 1 37 15 PM

### CODE REVIEW

File: docs/src/routes/docs/[...3]modules/core.md [hunk 1]
```diff
@@ -252,6 +252,20 @@ type i18nOptions = Record<Locale, i18n>
...
 Onboard is using the [ICU syntax](https://formatjs.io/docs/core-concepts/icu-syntax/) for formatting under the hood.
 
+For example, to update the connect interface language for Metamask, while giving a different message for other wallets, you can include the following: 
+
+```typescript
+i18n: {
+      en: {
+        connect: {
+          connectingWallet: {
+            paragraph: "{wallet, select, MetaMask {{wallet} can only present one account, so connect just the one account you want.} other {Please connect to all of your accounts in {wallet}.}}"
```
Comment (Adamj1232) [inline docs/src/routes/docs/[...3]modules/core.md:262]: Might be helpful to add more explanation as this is becomes a more functional interface. 
Would also be helpful to provide screenshots from the description along side.
Also need these docs in the core readme.

**Comment by leightkt** [comment]
Screenshot 2023-04-21 at 6 44 18 PM
Screenshot 2023-04-21 at 6 44 24 PM
Screenshot 2023-04-21 at 6 50 52 PM

**Comment by leightkt** [comment]
@Adamj1232 Updated Core readme and added screenshots! Lmk what you think!

File: packages/core/README.md [hunk 2]
```diff
@@ -257,6 +257,30 @@ type i18nOptions = Record<Locale, i18n>
...
+          connectingWallet: {
+            paragraph: "{wallet, select, MetaMask {{wallet} can only present one account, so connect just the one account you want.} other {Please connect to all of your accounts in {wallet}.}}"
+          }
+        }
+      }
+    }
+```
+
+MetaMask message:
+<img src="https://github.com/blocknative/web3-onboard/blob/develop/assets/custom-connect-1.png?raw=true" />
```
Comment (Adamj1232) [inline packages/core/README.md:275]: I dont see these images within the top level assets folder

File: packages/react/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/react",
-  "version": "2.8.0-alpha.2",
+  "version": "2.9.0-alpha.1",
```
Comment (Adamj1232) [inline packages/react/package.json:3]:
```suggestion
  "version": "2.8.0-alpha.2",
```

File: packages/vue/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/vue",
-  "version": "2.7.0-alpha.1",
+  "version": "2.8.0-alpha.1",
```
Comment (Adamj1232) [inline packages/vue/package.json:3]:
```suggestion
  "version": "2.7.0-alpha.2",
```

**Review by Adamj1232 — CHANGES_REQUESTED** [review]
just a couple quick updates of getting the screenshots in the top level assets folder for the README and the react/vue versions

File: packages/core/README.md (hunk 2 above)
Comment (leightkt) [reply packages/core/README.md:275]: ah yes- I only added them to the doc assets. Good catch- I'll update!

**Comment by leightkt** [comment]
@Adamj1232 Thanks for the version catches! Added the screenshots to the top level assets folder as well. Lmk if there's anything else I missed!

**Review by Adamj1232 — APPROVED** [review]

**Pull request merged**